packages/web-components/src/components/va-search/va-search.tsx

```diff
@@ -94,6 +154,31 @@ export class VaSearch {
    */
   @Prop() label: string = 'Search';
 
+  /**
+   * A boolean that controls whether suggestions are visible
+   */
+  @Prop() showSuggestions?: boolean;
+
+  /**
+   * An array of strings containing suggestions
+   */
+  @Prop() suggestions: any;
```

Comment:
We should try to avoid array props if we can since they won't be easy to use in the liquid templates.